lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -191,6 +192,67 @@ data class PendingBlockWithTransactionsResponse(
     override val parentHash: Felt,
 ) : GetBlockWithTransactionsResponse()
 
+sealed class GetBlockWithTransactionHashesResponse {
+    abstract val timestamp: Int
+    abstract val sequencerAddress: Felt
+    abstract val parentHash: Felt
+    abstract val transactionHashes: List<Felt>
+}
+
+@Serializable
+data class BlockWithTransactionHashesResponse(
+    @SerialName("status")
+    val status: BlockStatus,
+
+    // Block body
+
+    @SerialName("transactions")
+    override val transactionHashes: List<Felt>,
```

Comment:
shouldn't `transactions` also hold tx info? 
https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L1305

Reply:
We should be looking at `"BLOCK_WITH_TX_HASHES"` instead:
https://github.com/starkware-libs/starknet-specs/blob/d47828436349a163e0d02cc27d7dff98a1d5b28b/api/starknet_api_openrpc.json#L1371C13-L1371C13

Reply:
🙈 

sorry, checked the wrong endpoint